### DISCUSSION TITLE
Please help with css

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#27259] [Other Feature Feedback, Questions, & Ideas] [answered]
I need help with css on [my website](https://nit-nit.github.io) 😭😭

**Comment by jfullstackdev** [comment]
it's just the content of your `index.html`

![image](https://user-images.githubusercontent.com/47092464/181186003-efcbbb30-5600-408b-bdc5-ab2e4bb7d6c9.png)

and this is your site

![image](https://user-images.githubusercontent.com/47092464/181186081-c749a201-b295-4e3c-88d7-3274ebb8dd9e.png)

**Reply by NitinIsFit** [reply]
Yes, it mine 😭

**Comment by aayushchugh** [comment, marked as answer]
what should be expected behavior please explain it briefly

**Reply by NitinIsFit** [reply]
I want to make a card view, but i dont know how to make it responsive for desktop and other sizes screens 💀💀😭

**Reply by aayushchugh** [reply]
first you should add some more html in it there is no content just empty divs

**Reply by NitinIsFit** [reply]
Okay, i will tomorrow

**Reply by aayushchugh** [reply]
please mark this as answer too
1 already done after this one I will get badge

**Reply by aayushchugh** [reply]
thanks
